aiida/backends/tests/cmdline/commands/test_import.py

```diff
@@ -108,25 +109,64 @@ def test_import_old_local_archives(self):
-            self.assertIn(version, result.output, result.exception)
+            self.assertIsNone(result.exception, msg=result.output)
+            self.assertEqual(result.exit_code, 0, msg=result.output)
+            self.assertIn(version, result.output, msg=result.exception)
+            self.assertIn("{} after migration".format(archive), result.output, msg=result.exception)
+
+    def test_import_old_local_archive_no_migrate(self):
+        """ Test import of old local archives
+        Expected behavior: Return message in terminal and error code != 0
+        """
+        archive = 'export/migrate/export_v0.3.aiida'
+        version = '0.3'
+
+        options = ['--manual-migration', get_archive_file(archive)]
+        result = self.cli_runner.invoke(cmd_import.cmd_import, options)
+
+        self.assertIsNotNone(result.exception, msg=result.output)
+        self.assertNotEqual(result.exit_code, 0, msg=result.output)
+        self.assertIn(version, result.output, msg=result.exception)
+        self.assertNotIn("{} after migration".format(archive), result.output, msg=result.exception)
 
+    @unittest.skip("Reenable when issue #2426 has been solved (migrate exported files from 0.3 to 0.4)")
     def test_import_old_url_archives(self):
         """ Test import of old URL archives
-        Expected behavior: Return message in terminal and error code != 0
+        Expected behavior: Automatically migrate to newest version and import correctly.
+        TODO: Update 'url' to point at correct commit and file.
+        Now it is pointing to yakutovicha's commit, but when PR #2478 has been merged in aiidateam:develop,
+        url should be updated to point to the, essentially same, commit, but in aiidateam.
+        Furthermore, the filename should be changed from '_no_UPF.aiida' to '_simple.aiida'.
         """
-        url = "https://raw.githubusercontent.com/aiidateam/aiida_core/f685844b290ed23df254873df62da7162bde0f1b/"
+        url = "https://raw.githubusercontent.com/yakutovicha/aiida_core/f5fff1846a62051b898f13db67f5eef18892d5f4/"
         archive_path = "aiida/backends/tests/fixtures/export/migrate/"
-        archive = 'export_v0.1.aiida'
-        version = '0.1'
+        archive = 'export_v0.3_no_UPF.aiida'
+        version = '0.3'
 
         options = [url + archive_path + archive]
         result = self.cli_runner.invoke(cmd_import.cmd_import, options)
 
-        self.assertIsNotNone(result.exception, result.output)
-        self.assertNotEqual(result.exit_code, 0, result.output)
-        self.assertIn(version, result.output, result.exception)
+        self.assertIsNone(result.exception, msg=result.output)
+        self.assertEqual(result.exit_code, 0, msg=result.output)
+        self.assertIn(version, result.output, msg=result.exception)
+        self.assertIn("{} after migration".format(options[0]), result.output, msg=result.exception)
+
+    def test_import_old_url_archive_no_migrate(self):
```

Comment:
since you've tested the migrate/no-migrate already before, this test is not strictly necessary

Reply:
True. It shall be tossed aside.

Reply:
Done.